examples/controls_example/public/react_controls/timeslider_control/get_timeslider_control_factory.tsx

```diff
@@ -185,7 +185,7 @@ export const getTimesliderControlFactory = (
       const viewModeSubject =
         getViewModeSubject(controlGroupApi) ?? new BehaviorSubject('view' as ViewMode);
 
-      const defaultControl = initializeDefaultControlApi(initialState);
+      const defaultControl = initializeDefaultControlApi({ ...initialState, width: 'large' });
```

Comment:
The time slider should always have a width of `large`

Reply:
Can you turn your comment into a comment in the code please

Reply:
Good call - done in 34ac2265dae2c941cc1576ff47482ba1fa43506a